src/azure-cli/azure/cli/command_modules/keyvault/_format.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""Table transformer for keyvault commands"""
+
+
+def build_table_output(result, projection):
+
+    if not isinstance(result, list):
+        result = [result]
+
+    final_list = []
+
+    from collections import OrderedDict
+    for item in result:
+        def _value_from_path(each_item, path):
+            obj = each_item
+            try:
+                for part in path.split('.'):
+                    obj = obj.get(part, None)
+            except AttributeError:
+                obj = None
+            return obj or ' '
+
+        item_dict = OrderedDict()
```

Comment:
Since CPython 3.6, [`dict`](https://docs.python.org/3/library/stdtypes.html#dict) is the same as `OrderedDict`:

> Dictionary order is guaranteed to be insertion order. This behavior was an implementation detail of CPython from 3.6.

Reply:
changed